### ISSUE TITLE
Flash messages stay around with Turbolinks enabled

### ISSUE THREAD

**Issue by wout** [luckyframework/lucky#1093]
Disabling Turbolinks fixes the issue. Maybe flash messages are not cleared when performing XHR requests? Maybe related to #277?

**Comment by confact** [comment]
Notice this as well, if I click on links it persists. But if I do a real refresh (cmd+r / F5) it disappears. 

do @jwoertink or @paulcsmith know what the problem is here?

**Comment by jwoertink** [comment]
Nope. No clue 🤷‍♂️

**Comment by paulcsmith** [comment]
I need to look into it. But haven’t spent time on it yet. I need to get Dexter out so it removes all those warnings. Then I can look at this and others. 

May have something to do with https://github.com/luckyframework/lucky/issues/1064

Since flashes are stored in the session

**Comment by jwoertink** [comment]
Now that #1133 is merged, we should see if this is fixed.

**Comment by confact** [comment]
This seems to work now in Lucky 0.21.1

Do a last check and then I think you can close this :)

**Comment by jwoertink** [comment]
Yeah, this is looking good to me. If anyone runs in to this issue, we can reopen. Thanks @skojin !

**Comment by wout** [comment]
This is still not fixed for me. I'm using the latest master with the fixes for #1064 and flash messages still stick around when Turbolinks is enabled. It's easy to reproduce by adding the following action:

```cr
class SomeAction::Index < BrowserAction
  route do
    flash.failure = "I'm stubborn"
    redirect to: AnotherAction::Index
  end
end
```

If Turbolinks is enabled, the message stays there forever until a hard refresh, upon which it's shown one more time, and the next click on a link it's gone. With Turbolinks disabled, it works as expected.

**Comment by wout** [comment]
I can confirm this works as expected.
